distribution/lib/Standard/Base/0.0.0-dev/src/Network/Email/Email_Address.enso

```diff
@@ -0,0 +1,5 @@
+import project.Data.Text.Text
+
+type Email_Address
```

Comment:
- Why don't we put these types into `Email.enso` and expose just `Email` in `from ... import all`?
- then we could shorten these types to `type Address`
- and refer to it as `Email.Address`...